### ISSUE TITLE
open ondemand 2.0 updates

### ISSUE THREAD

**Issue by johrstrom** [NVIDIA/deepops#963]
Hey, I'm updating the playbooks for Open OnDemand 2.0 and thought I'd drop in and let you folks know about some updates.

I was going to tell you we're on galaxy now as `osc.open_ondemand`, but I see you use the git tags which is just as well.

We don't ship this file in 2.0. So you'll either have to write your own or just use the `user_map_match` that we'll start shipping with.
https://github.com/NVIDIA/deepops/blob/2879da8af1ddbc8297c541cb367073b2e738bff3/roles/ood-wrapper/tasks/server.yml#L205-L210

This works in debian's in v1.8.1+ of the role - you should be able to remove this and have everything work.  I'll continue to have ansible templates in 2.0.x, but I'd like to deprecate it just because it's duplicate work on my side with not a lot of benefit to the user.
https://github.com/NVIDIA/deepops/blob/2879da8af1ddbc8297c541cb367073b2e738bff3/roles/ood-wrapper/vars/ubuntu.yml#L6

**Comment by dholt** [comment]
Thanks Jeff!

**Comment by pedrosalgueiro** [comment]
I am installing Open OnDemand using DeepOps on a DGX A100 and run on this problem:

```
TASK [ood-wrapper : fix permissions on regex file] *****************************
fatal: [theHostname]: FAILED! => changed=false 
  msg: file (/opt/ood/ood_auth_map/bin/ood_auth_map.regex) is absent, cannot continue
  path: /opt/ood/ood_auth_map/bin/ood_auth_map.regex
  state: absent
```

Any help on how to solve it is appreciated.

**Comment by johrstrom** [comment]
Do you use and/or need that file? We've found most sites can get by with the `user_map_match` configuration instead.

Here are the release notes for the same. 
https://osc.github.io/ood-documentation/latest/release-notes/v2.0-release-notes.html#no-longer-providing-ood-auth-map-regex

**Comment by ajdecon** [comment]
Opened #1068 to remove this task from the DeepOps installation logic

**Comment by ksasagit** [comment]
This task, which was removed in 21.12, seems to be back in 22.01. In 22.01, the following error has occurred again.

> TASK [ood-wrapper : fix permissions on regex file] *****************************
> fatal: [theHostname]: FAILED! => changed=false 
>   msg: file (/opt/ood/ood_auth_map/bin/ood_auth_map.regex) is absent, cannot continue
>   path: /opt/ood/ood_auth_map/bin/ood_auth_map.regex
>   state: absent

**Comment by ajdecon** [comment]
Addressed by #1125
